R-packages/evalcast/man/compute_actual_vs_nominal_prob.Rd

```diff
@@ -0,0 +1,39 @@
+% Generated by roxygen2: do not edit by hand
+% Please edit documentation in R/calibration.R
+\name{compute_actual_vs_nominal_prob}
+\alias{compute_actual_vs_nominal_prob}
+\title{Compute calibration of a quantile forecaster}
+\usage{
+compute_actual_vs_nominal_prob(
+  score_card,
+  grp_vars = c("forecaster", "forecast_date", "ahead"),
+  avg_vars = c("geo_value")
+)
+}
+\arguments{
+\item{score_card}{tibble containing at least columns actual, quantile,
+value and any grouping or averaging variables named in the next arguments}
+
+\item{grp_vars}{character vector of named columns in the score_card at which
+average performance will be returned}
+
+\item{avg_vars}{character vector of named columns in the score_card over which
+averaging performance will be computed}
+}
+\value{
+A tibble containing grp_vars, nominal_prob (the claimed interval
+coverage), prop_below (the proportion of actual values falling below
+the forecast quantile) and prop_above (the proportion of
+actual values falling above the forecast quantile). All
+proportions are calculated for each available probability at each
+combination of grouping variables by averaging over any variables listed
+in avg_vars.
+}
+\description{
+Compute calibration of a quantile forecaster
```

Comment:
How is this different from compute_calibration?

Reply:
It's the same. That function was in the old version but does the same thing.

Reply:
Should we get rid of it then?